### PR TITLE
change dockerfile directory

### PR DESCRIPTION
**Pull request by anwilkie**
# What and why?
The dockerfile needed to be moved to its own folder in _infra in order to make it consistent with the other repos.

A few pipenv dependencies were updated, too, and some vulnerability warnings were ignored like they are on other repos (and number 70624 was still cropping up even though flask-cors wouldn't update any further).

# How to test?
Check GA build

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-313)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ build-kubernetes:
 	docker build -f _infra/docker/Dockerfile .
 
 lint:
-	pipenv check ./application ./tests -i 51457
+	pipenv check ./application ./tests -i 70612 -i 71064 -i 70624
```
Comment (arroyoAle) [inline Makefile:11]: There is an update for 71064 but that cane be another card

**Review by arroyoAle — APPROVED** [review]
Looks good and works fine. Just a small comment that would be a different card

File: Pipfile [hunk 2]
```diff
@@ -25,6 +25,7 @@ pycryptodome = "*"
 requestsdefaulter = "*"
 google-cloud-storage = "*"
 google-cloud-pubsub = "*"
+urllib3 = "*"
```
Comment (matthew-robinson-ons) [inline Pipfile:28]: As with secure message we shouldn't be importing urllib3 but updating the package that manages the dependency

**Review by matthew-robinson-ons — CHANGES_REQUESTED** [review]
As with secure message urllib3 needs looking at but the docker changes look good

File: _infra/docker/Dockerfile
Comment (arroyoAle) [inline _infra/docker/Dockerfile:1]: This dockerfile in the root directory is used in the `docker-compose up -d` command in the readme

**Review by arroyoAle — CHANGES_REQUESTED** [review]
I noticed this while working on another card but if we do not have the image already built locally, the docker compose command will fail as it is expecting a docker configuration file

File: Pipfile (hunk 2 above)
Comment (anwilkie) [reply Pipfile]: Fixed.

File: _infra/docker/Dockerfile
Comment (anwilkie) [reply _infra/docker/Dockerfile:1]: I added the directory to the docker-compose.yml file, so that the docker compose command uses the new file location.

**Pull request closed without merging**